allennlp/models/semantic_parsing/atis/atis_semantic_parser.py

```diff
@@ -0,0 +1,592 @@
+import logging
```

Comment:
We should not have the ATIS db checked into the repo like it is (commenting here, because I can't comment on the db file).  Instead, put it somewhere on S3 and use a URL and `cached_path` to access it.  I can help you put it on S3 if you have trouble.

Reply:
Added it as a dataset here: https://s3-us-west-2.amazonaws.com/allennlp/datasets/atis/atis.db.